pkgs/development/python-modules/jupyter_core/fix-search-path.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/jupyter_core/command.py b/jupyter_core/command.py
+index b9c60d2..d30e2fd 100644
+--- a/jupyter_core/command.py
++++ b/jupyter_core/command.py
+@@ -166,7 +166,7 @@ def _path_with_self():
+     else:
+         path_list.append(bindir)
+ 
+-    scripts = [sys.argv[0]]
++    scripts = [sys.argv[0],sys.executable]
```

Comment:
I'm a bit confused here; perhaps you can fill in something I'm missing.

In a comment, you say that when you run into the issue this is meant to fix:
> `sys.argv[0]` is pointing to the raw python interpreter instead of the wrapper.

But `sys.executable` is itself meant to be the path to the Python interpreter. (That's its documented meaning upstream, at least.)

What path is `sys.executable` pointing to in the situation where you're seeing the issue, and how does it relate to the path pointed to by `sys.argv[0]`?

Reply:
I'm not too sure on how the those are actually evaluated, but `sys.argv[0]` doesn't point to `sys.executable` since I had to make this in the first place.

The actual python script in the `python-env` is a wrapper that calls the python interpreter, so that's probably why `sys.argv[0]` evaluates to the interpreter while `sys.executable` refers to the script that started it.